manager/system_handler.py

```diff
@@ -444,7 +445,7 @@ def _full_query(rh_account_id):
 
         return (SystemPlatform
                 .select(*selectables)
-                .where(system_is_active(rh_account_id=rh_account_id, deleted=False, edge=False, opt_out=None, stale=None))
+                .where(system_is_active(rh_account_id=rh_account_id, deleted=False, edge=edge_feature_arg(), opt_out=None, stale=None))
```

Comment:
This should be `edge=None` to allow listing all systems. Listing is later filtered by host type filter. What am I missing?